### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by afeezaziz** [jmiller656/DiscoGAN-Tensorflow#2]
Hi @jmiller656 , I am playing around with DiscoGAN and thanks for letting me experimenting with it using your examples. 

Are you looking forward to make this project more stable or are you planning this as a one-off project?

**Comment by jmiller656** [comment]
Hey, thanks for reaching out. I do have quite a bit more work planned for this project in order to make things a bit more stable. Some of the changes I have in mind are:

- Refactoring to make this more user friendly (no more directly editing source code to work on your own examples)
- Dynamic image size, right now the net is stuck with using and generating 64x64 images
- Interchangeable operations inside the net (ex. using different loss functions, the paper mentions a few and it might be fun to experiment with others)

Also, I plan on training on the actual sets that were used in the paper and adding the results to the readme
